specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-03-01/orbital.json

```diff
@@ -0,0 +1,2228 @@
+          }
+        },
+        "links": {
+          "type": "array",
+          "x-ms-identifiers": [],
+          "items": {
+            "$ref": "#/definitions/ContactProfileLink"
+          },
+          "description": "Links of the Contact Profile"
+        }
+      },
+      "description": "List of Contact Profile Resource Properties."
+    },
+    "ContactProfileListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContactProfile"
+          },
+          "description": "A list of contact profile resources in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for the ListContactProfiles API service call."
+    },
+    "ContactProfileLinkChannel": {
+      "type": "object",
+      "required": [
+        "name",
+        "centerFrequencyMHz",
+        "bandwidthMHz",
+        "endPoint"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Channel name"
+        },
+        "centerFrequencyMHz": {
+          "type": "number",
+          "description": "Center Frequency in MHz"
+        },
+        "bandwidthMHz": {
+          "type": "number",
+          "description": "Bandwidth in MHz"
+        },
+        "endPoint": {
+          "$ref": "#/definitions/EndPoint"
+        },
+        "modulationConfiguration": {
+          "type": "string",
+          "description": "Configuration for modulation"
+        },
+        "demodulationConfiguration": {
+          "type": "string",
+          "description": "Configuration for demodulation"
+        },
+        "encodingConfiguration": {
+          "type": "string",
+          "description": "Configuration for encoding"
+        },
+        "decodingConfiguration": {
+          "type": "string",
+          "description": "Configuration for decoding"
+        }
+      },
+      "description": "Contact Profile Link Channel"
+    },
+    "ContactProfileLink": {
+      "type": "object",
+      "required": [
+        "name",
+        "polarization",
+        "direction",
+        "channels"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Link name"
+        },
+        "polarization": {
+          "type": "string",
+          "description": "polarization. eg (RHCP, LHCP)",
+          "enum": [
+            "RHCP",
+            "LHCP",
+            "linearVertical",
+            "linearHorizontal"
+          ],
+          "x-ms-enum": {
+            "name": "polarization",
+            "modelAsString": true
+          }
+        },
+        "direction": {
+          "type": "string",
+          "description": "Direction (uplink or downlink)",
+          "enum": [
+            "uplink",
```

Comment:
Based on a live test, the enum of `direction` should be `Uplink` and `Downlink`.

Here're the requests traces: https://gist.github.com/ms-henglu/9f9d848881bad9c6347fc97543341cc1